webapp/chart.js

```diff
@@ -799,7 +800,23 @@ function chart_init(graph, wnd, affichwnd, with_interval, with_interval2, show_i
                 points_list[index].enqueue([line_id, x, y]);
             }else{
                 var values = get_points_for_data(data, data[8], data[10]);
-                points_list[index].enqueue([line_id, values]);
+                cmatview_counter++; // to count lines from log
+                if(cmatview_counter == 1){
+                     //Only add points of line 1, so that no delay in chart appearance)
+                     points_list[index].enqueue([line_id, values]);
+                }else if(cmatview_counter < 16){
+                     //Only add points of line which are multiple of 5, till 15 like 5 10 15 (this is to reduce load on browser)
+                    var count = cmatview_counter % 5;
+                    if(count == 0){
+                        points_list[index].enqueue([line_id, values]);
+                    }
+                }else{
+                    //Only add points of line which are multiple of 10 but after 16 like 20 30 ... (this is to reduce load on browser)
+                    var count = cmatview_counter % 10;
```

Comment:
```suggestion
                    var count = cmatview_counter[index] % 25;
```
Try increasing gap to 25 for the lines later. Add index as appropriate.